### ISSUE TITLE
Use contrasting colours when displaying Github labels

### ISSUE THREAD

**Issue by tadzik** [matrix-org/matrix-hookshot#432]
**Is your feature request related to a problem? Please describe.**
When hookshot notifies a room about a label changing, some colour combinations make it hard to read.

**Describe the solution you'd like**
Hookshot should pick a contrasting foreground colour so that the label can be read easily (maybe with something like https://www.npmjs.com/package/get-best-contrast-color)

**Describe alternatives you've considered**
Alternatively, a single square with the right colour could be used, and the label name could be kept in the default colour.

**Additional context**

![image](https://user-images.githubusercontent.com/86378/182142680-c8163c2f-c748-473b-9b5c-9e73d5285c6d.png)

**Comment by Half-Shot** [comment]
Yeah, I believe it's just quite poor at it. https://github.com/matrix-org/matrix-hookshot/blob/main/src/format_util.rs#L93-L104 is used to determine the contrast.
